selene/core/entity.py

```diff
@@ -174,26 +176,31 @@ def __call__(self) -> WebElement:
     # --- WaitingEntity --- #
 
     @property
-    def wait(self) -> Wait[E]:  # todo:  will not it break code like browser.with_(timeout=...)?
+    def wait(self) -> Wait[E]:
+        # todo:  will not it break code like browser.with_(timeout=...)?
         # todo: fix that will disable/break shared hooks (snapshots)
         # return Wait(self,  # todo:  isn't it slower to create it each time from scratch? move to __init__?
         #             at_most=self.config.timeout,
         #             or_fail_with=pipe(
         #                 Element._log_webelement_outer_html_for(self),
         #                 self.config.hook_wait_failure))
         if self.config.log_outer_html_on_failure:
-            '''
+            """
             todo: remove this part completely from core.entity logic
                   move it to support.shared.config
-            '''
-            return super().wait.or_fail_with(pipe(
-                Element._log_webelement_outer_html_for(self),
-                super().wait.hook_failure))
+            """
```

Comment:
even with disabled string quote auto-formatting "black" formats docstrings with double quotes way. I don't think that it's a problem. 🙂

Reply:
yes, I can leave with that)